src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs

```diff
@@ -140,25 +140,8 @@ public ExecuteMultipleResponse UpdateStateAndStatusForEntityInBatch(EntityCollec
         }
 
         /// <inheritdoc/>
-        public void ImportWordTemplate(string filePath)
+        public void ImportWordTemplate(FileInfo fileInfo, string entityLogicalName, OptionSetValue templateType, string filePath)
```

Comment:
Technically a breaking change to the method signature although I doubt it's being called other than internally at the moment. Generally the sort of change we would avoid making.

Reply:
Ah yes, very good point, @ewingjm. I'll watch out for this next time but agree it's something we can forgive this time. Do you think we should increase the major version accordingly?

Reply:
We're still v0.x so I think it's OK to make the occasional small breaking change. In this case it does look like the logic made sense to be moved elsewhere as having the `CrmServiceAdapter` was responsible for too much.

Reply:
I'd just let it increment the patch version as normal 👍🏻

Reply:
Reason that was made was to make sure the sole responsibility of the CRMServiceAdapter is to interact with the service layer and nothing else. before it was calling Utilities to strip entitynames and etc which as far as Service Adapter is concerned should be passed in as params.